web/js/tasks.js

```diff
@@ -699,6 +699,43 @@ var TaskPanel = Ext.extend(Ext.Panel, {
         this.teleworkCheckBox.setValue((this.taskRecord.data['telework']=='true'));
         this.onsiteCheckBox.setValue((this.taskRecord.data['onsite']=='true'));
 
+        bottomBoxTaskItems = [
+                        new Ext.form.Label({text: 'Task type', style: 'display:block; padding:5px 2px 5px 2px'}),
+                        this.taskTypeComboBox
+        ]
+
+        if (menuCoordination == true ) {
+            bottomBoxTaskItems.push(
+               new Ext.form.Label({text: 'TaskStory', style: 'display:block; padding:5px 2px 5px 2px'}),
+              this.taskStoryComboBox
+            )
+        }
+
+        bottomBoxItems = [
+                new Ext.Container({
+                    width: 186,
+                    layout: 'hbox',
+                    items:[
+                        this.deleteButton,
+                        this.cloneButton,
+                        this.createTemplateButton,
+                    ]
+                }),
+                new Ext.Container({
+                    width: 120,
+                    layout: 'hbox',
+                    items:[
+                        this.teleworkCheckBox,
+                        this.onsiteCheckBox
+                    ]
+                }),
+                new Ext.Container({
+                    width: 460,
```

Comment:
We should probably use a different width here when coordination is disabled. With my usual browser and window size, it looks like this:
![screenshot from 2017-09-04 12-08-47](https://user-images.githubusercontent.com/2900877/30022121-36ec66c8-916a-11e7-83a0-05ad43f61129.png)